### PR TITLE
Remove pg from evm tests

### PR DESCRIPTION
**Pull request by DylanTinianov**
Remove `core/services/pg` from evm txStore tests

### CODE REVIEW

File: core/chains/evm/txmgr/test_helpers.go [hunk 1]
```diff
@@ -145,7 +143,7 @@ func (c *MockConfig) FinalityTagEnabled() bool          { return c.finalityTagEn
 func (c *MockConfig) RPCDefaultBatchSize() uint32       { return c.RpcDefaultBatchSize }
 
 func MakeTestConfigs(t *testing.T) (*MockConfig, *TestDatabaseConfig, *TestEvmConfig) {
-	db := &TestDatabaseConfig{defaultQueryTimeout: pg.DefaultQueryTimeout}
+	db := &TestDatabaseConfig{defaultQueryTimeout: 10 * time.Second}
```
Comment (patrickhuie19) [inline core/chains/evm/txmgr/test_helpers.go:146]: should this be a named constant living in common?

File: core/chains/evm/txmgr/evm_tx_store_test.go [hunk 2]
```diff
@@ -1214,25 +1213,16 @@ func TestORM_LoadEthTxesAttempts(t *testing.T) {
 		etx := mustInsertConfirmedMissingReceiptEthTxWithLegacyAttempt(t, txStore, 3, 9, time.Now(), fromAddress)
 		etx.TxAttempts = []txmgr.TxAttempt{}
 
-		q := pg.NewQ(db, logger.Test(t), cfg.Database())
-
 		newAttempt := cltest.NewDynamicFeeEthTxAttempt(t, etx.ID)
 		var dbAttempt txmgr.DbEthTxAttempt
```
Comment (prashantkumar1982) [inline core/chains/evm/txmgr/evm_tx_store_test.go:1217]: This var can be deleted now, it is unused

File: core/chains/evm/txmgr/evm_tx_store_test.go [hunk 3]
```diff
@@ -1214,25 +1213,16 @@ func TestORM_LoadEthTxesAttempts(t *testing.T) {
...
-			return nil
-		})
+
+		err := txStore.InsertTxAttempt(ctx, &newAttempt)
 		require.NoError(t, err)
+
+		err = txStore.LoadTxesAttempts(ctx, []*txmgr.Tx{&etx})
+		require.NoError(t, err)
+		assert.Len(t, etx.TxAttempts, 2)
 		// also check after postgres transaction is committed
```
Comment (prashantkumar1982) [inline core/chains/evm/txmgr/evm_tx_store_test.go:1226]: The next 4 lines are exact duplicates of previous 3 lines.
I think now we can delete the redundant checks. Previously it was done in addition to the above pg tx, which you now removed

File: core/chains/evm/txmgr/test_helpers.go (hunk 1 above)
Comment (jmank88) [reply core/chains/evm/txmgr/test_helpers.go:146]: I don't think that we need to couple all of the modules together, but this does seem worth a constant somewhere within EVM.

File: core/chains/evm/txmgr/evm_tx_store_test.go [hunk 4]
```diff
@@ -1214,25 +1213,16 @@ func TestORM_LoadEthTxesAttempts(t *testing.T) {
 		etx := mustInsertConfirmedMissingReceiptEthTxWithLegacyAttempt(t, txStore, 3, 9, time.Now(), fromAddress)
 		etx.TxAttempts = []txmgr.TxAttempt{}
 
-		q := pg.NewQ(db, logger.Test(t), cfg.Database())
-
 		newAttempt := cltest.NewDynamicFeeEthTxAttempt(t, etx.ID)
 		var dbAttempt txmgr.DbEthTxAttempt
 		dbAttempt.FromTxAttempt(&newAttempt)
-		err := q.Transaction(func(tx pg.Queryer) error {
```
Comment (jmank88) [inline core/chains/evm/txmgr/evm_tx_store_test.go:1222]: Why is it OK to drop the tx?

File: core/chains/evm/txmgr/test_helpers.go (hunk 1 above)
Comment (DylanTinianov) [reply core/chains/evm/txmgr/test_helpers.go]: Added constant to `evm/txmgr/utils`

File: core/chains/evm/txmgr/evm_tx_store_test.go (hunk 4 above)
Comment (DylanTinianov) [reply core/chains/evm/txmgr/evm_tx_store_test.go:1222]: Updated to keep in a tx.

**Review by jmank88 — APPROVED** [review]

**Review by patrickhuie19 — APPROVED** [review]

**Review by prashantkumar1982 — APPROVED** [review]

**Pull request merged**